packages/expo-dev-menu/ios/Tests/DevMenuAppInstanceTest.swift

```diff
@@ -55,10 +54,11 @@ class DevMenuAppInstanceTest: QuickSpec {
       let extraModules = appInstance.extraModules(for: mockedBridge)
 
       expect(extraModules).toNot(beNil())
-      expect(extraModules?.count).toNot(equal(7))
+      expect(extraModules?.count).toNot(equal(8))
```

Comment:
I compared the results from when this test was passing and the difference was before we had "MockedRNCSafeAreaProvider" whereas now we have the two real modules.

@lukmccall -- curious why this statement is a "not equal" rather than just looking for an exact match? I'd like to at least add a comment since it was unclear how to correctly fix this test without checking out an old commit.

Reply:
I think it was a mistake - it should be an exact match.